frontend/src/components/Title.tsx

```diff
@@ -0,0 +1,41 @@
+import classes from './Title.module.css';
+
+type Level = "small" | "medium" | "large"
+
+interface TitleProps {
+    level: Level;
+}
+
+function Title({level} : TitleProps) {
+    let size : number;
+    switch(level) {
+        case "small":
+            size = 1;
+            break;
+        case "medium":
+            size = 1.5;
+            break;
+        case "large":
+            size = 2;
+            break;
```

Comment:
wow - these sizes are really dynamic 
![Uploading Screenshot 2024-07-23 at 11.11.12 pm.png…]()

Reply:
wow thanks for the awesome picture